README.md

```
@@ -174,18 +153,17 @@ Once you've set up the chat and added the necessary components, you can start th
 ```js
 let response = chat.run({
-  model: "gemini-1.5-pro-002", // Optional: set the model to use
-  temperature: 0.5 // Optional: set response creativity
+  model: "gemini-2.5-flash", // Optional: set the model to use
+  temperature: 0.5, // Optional: set response creativity
   function_call: "getWeather" // Optional: force the first API response to call a function
 });
 
 console.log(response);
+
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)